src/all/pandachaika/src/eu/kanade/tachiyomi/extension/all/pandachaika/PandaChaikaDto.kt

```diff
@@ -8,21 +8,21 @@ import java.util.Date
 import java.util.Locale
 
 val dateReformat = SimpleDateFormat("EEEE, d MMM yyyy HH:mm (z)", Locale.ENGLISH)
-fun filterTags(include: String = "", exclude: List<String> = emptyList(), tags: List<String>): String {
+fun filterTags(include: String = "", exclude: List<String> = emptyList(), tags: List<String>): String? {
     return tags.filter { it.startsWith("$include:") && exclude.none { substring -> it.startsWith("$substring:") } }
         .joinToString {
             it.substringAfter(":").replace("_", " ").split(" ").joinToString(" ") { s ->
                 s.replaceFirstChar { sr ->
                     if (sr.isLowerCase()) sr.titlecase(Locale.getDefault()) else sr.toString()
                 }
             }
-        }
+        }.takeIf { it.isNotBlank() }
 }
 fun getReadableSize(bytes: Double): String {
     return when {
-        bytes >= 300 * 1024 * 1024 -> "${"%.2f".format(bytes / (1024.0 * 1024.0 * 1024.0))} GB"
-        bytes >= 100 * 1024 -> "${"%.2f".format(bytes / (1024.0 * 1024.0))} MB"
-        bytes >= 1024 -> "${"%.2f".format(bytes / (1024.0))} KB"
+        bytes >= 300 * 1000 * 1000 -> "${"%.2f".format(bytes / (1000.0 * 1000.0 * 1000.0))} GB"
+        bytes >= 100 * 1000 -> "${"%.2f".format(bytes / (1000.0 * 1000.0))} MB"
+        bytes >= 1000 -> "${"%.2f".format(bytes / (1000.0))} KB"
```

Comment:
Since you're actually using 10-base values here (as opposed to 2-base values, as in Koharu).

-- you should use `kB` and not `KB`. See https://en.wikipedia.org/wiki/Kilobyte#Definitions_and_usage , where it's pointed out that 2-base values use `KB` and 10-base values use `kB`